dev/ci/user-overlays/14711-herbelin-master+univs-ctx-and-binders-centralized.sh

```diff
@@ -0,0 +1,6 @@
+overlay equations https://github.com/herbelin/Coq-Equations master+adapt-coq-pr14711-univs-ctx-and-binders-centralized 14711 master+univs-ctx-and-binders-centralized
+overlay elpi https://github.com/herbelin/coq-elpi coq-master+adapt-coq-pr14711-univs-ctx-and-binders-centralized 14711 master+univs-ctx-and-binders-centralized
+overlay paramcoq https://github.com/herbelin/paramcoq master+adapt-coq-pr14711-univs-ctx-and-binders-centralized 14711 master+univs-ctx-and-binders-centralized
+overlay mtac2 https://github.com/herbelin/Mtac2 master+adapt-coq-pr14711-univs-ctx-and-binders-centralized 14711 master+univs-ctx-and-binders-centralized
+overlay rewriter https://github.com/herbelin/Mtac2 master+adapt-coq-pr14711-univs-ctx-and-binders-centralized 14711 master+univs-ctx-and-binders-centralized
```

Comment:
```suggestion
overlay rewriter https://github.com/herbelin/rewriter master+adapt-coq-pr14711-univs-ctx-and-binders-centralized 14711 master+univs-ctx-and-binders-centralized
```
X_X
I'm pushing this change